website_snippet_markdown/__manifest__.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Onestein (<http://www.onestein.eu>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    'name': 'Website Markdown Snippet',
+    'category': 'Website',
+    'version': '10.0.1.0.0',
+    'author': 'Onestein', 'Odoo Community Association (OCA)",
```

Comment:
`'Onestein', 'Odoo Community Association (OCA)"` will not work, please use `"Onestein, Odoo Community Association (OCA)"` instead.